vignettes/Age_metaanalysis_vignette.Rmd

```diff
@@ -0,0 +1,348 @@
+---
+title: "Meta-analysis of age-related microbial species using cMD3.0"
+author: 
+  - name: Davide Golzato
+    affiliation:
+    - Università degli Studi di Trento
+    email:
+    - davide.golzato@unitn.it
+  - name: Paolo Manghi
+    affiliation:
+    - Università degli Studi di Trento
+    email:
+    - paolo.manghi@unitn.it
+    
+output: 
+  BiocStyle::html_document:
+    self_contained: yes
+    toc: true
+    toc_float: true
+    toc_depth: 2
+    code_folding: show
+date: "`r doc_date()`"
+package: "`r pkg_ver('curatedMetagenomicData')`"
+vignette: >
+  %\VignetteIndexEntry{Put an analysis here}
+  %\VignetteEngine{knitr::rmarkdown}
+  %\VignetteEncoding{UTF-8}  
+---
+
+
+<style>
+pre {
+  overflow-x: auto;
+}
+pre code {
+  word-wrap: normal;
+  white-space: pre;
+  max-width: 1000px
+}
+</style>
+
+```{r setup, include = FALSE}
+library("knitr")
+library(kableExtra)
+
+options(width = 120)
+knitr::opts_chunk$set(
+    collapse = TRUE,
+    comment = "#>",
+    crop = NULL, ## Related to https://stat.ethz.ch/pipermail/bioc-devel/2020-April/016656.html,
+    tidy.opts=list(width.cutoff=120)
+)
+```
+
+
+# Meta-analysis of age-related microbial species using data from curatedMetagenomicData
+
+```{r, message=FALSE}
+library(curatedMetagenomicData)
+library(tidyverse)
+library(SummarizedExperiment)
+library(TreeSummarizedExperiment)
+```
+
+
+## Retrieval of datasets from the cMD3 meeting a series of requirements
+```{r}
+#Filter on Age, body_site, study_condition and remove samples missing BMI
+  metadata <- curatedMetagenomicData::combined_metadata  %>% 
```

Comment:
I think we should use `curatedMetagenomicData::sampleData` here and in the sex_metaanalysis_vignette on line 68 because `combined_metadata` is deprecated. The data is the same.